src/content/docs/style-guide/word-choice/pricing-language-guidelines.mdx

```diff
@@ -45,15 +45,11 @@ Some notes on style and formatting for pricing-related language:
 * "usage-based." Our newer pricing model can be referred to as “usage-based” or “consumption-based,” but “usage-based” is the most commonly used and more preferred term.
 * Editions:
   * Our pricing editions are not "plans" or "models" or "tiers"; they are "editions".
+  * We don't refer to our [free tier](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#free) an edition, but it is basically an edition, because it has different limits than Standard edition. To be on Standard edition means that a customer is paying and not on the free tier. 
```

Comment:
```suggestion
  * We don't refer to our [free tier](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#free) as an edition, but it is basically an edition, because it has different limits than Standard edition. To be on Standard edition means that a customer is paying and not on the free tier. 
```